packages/core/src/interpreter.ts

```diff
@@ -409,16 +406,33 @@ export class Interpreter<
             State.from(initialState, this.machine.context)
           );
 
-    this.scheduler.initialize(() => {
-      this.update(resolvedState, initEvent as SCXML.Event<TEvent>);
```

Comment:
i suspect that the `initEvent` is still a thing, right? where is it handled now?

Reply:
It comes from here:

```ts
export function resolveMicroTransition<
  TContext extends MachineContext,
  TEvent extends EventObject
>(
  machine: StateMachine<TContext, TEvent>,
  transitions: Transitions<TContext, TEvent>,
  currentState: State<TContext, TEvent, any>,
  _event: SCXML.Event<TEvent> = initEvent as SCXML.Event<TEvent> // <<< here
): State<TContext, TEvent, any> {
```

The machine has this:
```ts
    const nextState = resolveMicroTransition(
      this,
      [],
      this.preInitialState,
      undefined // <<< resolves to initEvent
    );
```